node/src/daemon/setup_reporter.rs

```diff
@@ -2413,9 +2413,18 @@ mod tests {
             "config_file_not_specified_but_exists",
         );
         {
-            let config_file_path = data_directory.join("config.toml");
-            let mut config_file = File::create(config_file_path).unwrap();
-            config_file.write_all(b"gas-price = \"10\"\n").unwrap();
+            let config_file_path = data_directory
+                .join("MASQ")
+                .join("polygon-mainnet")
+                .join("config.toml");
+            let created_dir = create_dir_all(data_directory.join("MASQ").join("polygon-mainnet"));
```

Comment:
With regards to the fact that the only error we can have here are wrongly set rights for creating folders, you can just use a function to unwrap the result but maybe .expect() because you can write a special message there like something about why it failed. 

So it will replace the if statement below.